plugins/doc_fragments/aws.py

```diff
@@ -65,7 +73,9 @@ class ModuleDocFragment(object):
     C(AWS_ACCESS_KEY_ID) or C(AWS_ACCESS_KEY) or C(EC2_ACCESS_KEY),
     C(AWS_SECRET_ACCESS_KEY) or C(AWS_SECRET_KEY) or C(EC2_SECRET_KEY),
     C(AWS_SECURITY_TOKEN) or C(EC2_SECURITY_TOKEN),
-    C(AWS_REGION) or C(EC2_REGION)
+    C(AWS_REGION) or C(EC2_REGION),
+    C(AWS_PROFILE) or C(AWS_DEFAULT_PROFILE),
+    C(AWS_CA_BUNDLE)
```

Comment:
I wish that this custom list would and the environment var precedence code in module_utils/ec2 could be removed entirely, because it would be nice to just let boto3 do it. But your change look good in terms of the current pattern.

Reply:
I have vague memories that simply letting boto3 handle it can have strange side effects if you don't run with `connection: local`...